tests/_utils/ui/editableui/framed/framededitableuiiframeview.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import IframeView from '/ckeditor5/ui/iframe/iframeview.js';
+import FramedEditableUIView from './framededitableuiview.js';
+
+/**
+ * The basic implementation of an {@link ui.iframe.IframeView IframeView}-based
+ * {@link ui.editableUI.EditableUIView}.
+ *
+ * @memberOf ui.editableUI.iframe
+ * @extends ui.iframe.IframeView
+ */
+export default class FramedEditableUIIframeView extends IframeView {
+	/**
+	 * Creates a new instance of the {@link ui.iframe.IframeView IframeView}–based
+	 * {@link ui.editableUI.EditableUIView EditableUIView}.
+	 *
+	 * @param {utils.Observable} [model] (View)Model of this View.
+	 * @param {utils.Locale} [locale] The {@link ckeditor5.Editor#locale editor's locale} instance.
+	 */
+	constructor( model, locale ) {
+		super( model, locale );
+
+		const bind = this.attributeBinder;
+
+		this.template.attributes.class.push( 'ck-editor__editable-iframe' );
+		this.template.attributes.style = [
+			'width:',
+			bind.to( 'width', ( w ) => `${w}px` ),
```

Comment:
This is bad... We need to find a better solution, which would be an object format for styles. We can't rewrite all styles every time. I'll create a followup for this.